plugins/all-modules-page/build.gradle.kts

```diff
@@ -28,4 +28,5 @@ dependencies {
     testImplementation(projects.core.testApi)
     testImplementation(platform(libs.junit.bom))
     testImplementation(libs.junit.jupiter)
+    testRuntimeOnly(project(path = ":subprojects:analysis-kotlin-descriptors", configuration = "shadow"))
```

Comment:
> Do our plugins not work with K2?

Some should work. 
But for `kotlin-as-java`  and `javadoc`  plugins, symbols do not work. `kotlinToJavaService, inheritanceBuilder` are not implemented here. 

> Will the tests fail if the analysis here is switched? 

The tests should not fail since they do not check the Documentable model, but it is only for this plugin.